devenv/sync.py

```diff
@@ -194,19 +193,27 @@ def main(context: dict[str, str]) -> int:
                 {
                     "NODE_ENV": "development",
                 },
-            ),
+            )
+        )
+    procs.append(
+        (
+            "python dependencies (2/4)",
             (
-                "python dependencies (2/4)",
-                (
-                    "pip",
-                    "uninstall",
-                    "-qqy",
-                    "djangorestframework-stubs",
-                    "django-stubs",
-                ),
-                {},
+                "pip",
+                "uninstall",
+                "-qqy",
+                "djangorestframework-stubs",
```

Comment:
@asottile-sentry do we still need this?

Reply:
ah yeah we can remove this now that we've upgraded python versions

Reply:
@wedamija ^